wp-theme-2018/shortcodes/epfl_memento/utils.php

```diff
@@ -60,4 +60,27 @@ function is_inscription_required($invitation) {
     return ($invitation === "Registration required");
 }
 
+/** 
+ * Get visual url of event
+ */
+function get_visual_url($event, $memento_name) {
+
+    $visual_url = "";
 
+    if ($event->academic_calendar_category == null) {
```

Comment:
Je suis plus fan d'un empty() ou un isset() ici
https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/
```suggestion
    if (empty($event->academic_calendar_category)) {
```